test/sanityQueryService.test.js

```diff
@@ -293,4 +293,22 @@ describe('Sanity Queries', function () {
         const response = await fetchScheduledReleases('drumeo', {});
         expect(response.length).toBeGreaterThan(0);
     });
+
+    test('fetchAll-GroupBy-Genre', async () => {
+        let response = await fetchAll('drumeo', 'solo',{groupBy: 'genre'});
+        console.log(response);
+        expect(response.entity[0].web_url_path).toContain('/drumeo/genres/');
+    });
+
+    test('fetchAll-GroupBy-Artists', async () => {
+        let response = await fetchAll('drumeo', 'song',{groupBy: 'artist'});
+        console.log(response);
```

Comment:
Also nitpicking and I'm not sure if I'm the only one with this opinion, but I would recommend commenting out the `console.log` calls since it makes some noise that can make it harder to track test output.

Reply:
I commented out the console.log calls